test/integration_tests/test_load_executor.py

```diff
@@ -35,7 +35,7 @@ def tearDown(self):
         os.remove('dummy.avi')
 
     # integration test
-    @unittest.skip("we need drop functionality before we can enable")
+    # @unittest.skip("we need drop functionality before we can enable")
```

Comment:
We need to wait for drop. I have checked locally it works.

Reply:
Drop is only needed for "Create". By the current way load is implemented, Drop is not required.

Reply:
Yup sure got it.